### PR TITLE
move translations/popularity to app_translations_by_popularity/view

### PR DESCRIPTION
**Pull request by dannyroberts**
This decouples Applications from the translation app in preparation for apps doctype migration

### CODE REVIEW

File: corehq/couchapps/app_translations_by_popularity/views/view/map.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+function (doc) {
```
Comment (dimagimon) [inline corehq/couchapps/app_translations_by_popularity/views/view/map.js]: Missing name in function declaration.

File: corehq/apps/translations/_design/views/popularity/map.js [hunk 2]
```diff
@@ -1,10 +0,0 @@
-function (doc) {
-    if ((doc.doc_type === 'Application' && doc.copy_of === null) || doc.doc_type === 'TranslationDoc') {
```
Comment (dannyroberts) [inline corehq/apps/translations/_design/views/popularity/map.js]: `TranslationDoc` was never a doc_type (ever). I've simplified this in the new view to reflect that

Comment (esoergel) [inline corehq/apps/translations/_design/views/popularity/map.js]: :rage4:

**Comment by esoergel** [comment]
I'm signed off - gtm when reindex is convenient.

File: corehq/apps/translations/_design/views/popularity/map.js (hunk 2 above)
Comment (czue) [inline corehq/apps/translations/_design/views/popularity/map.js]: classic.

**Pull request merged**